chart/kubeapps/templates/kubeops-deployment.yaml

```diff
@@ -63,6 +63,12 @@ spec:
             {{- if .Values.kubeops.QPS }}
             - --qps={{ .Values.kubeops.QPS }}
             {{- end }}
+            {{- if .Values.kubeops.ns-header-name }}
```

Comment:
You didn't test this did you? To shame!! 😆 

```
{{ index .Values.kubeops "ns-header-name" }}
```

https://stackoverflow.com/questions/63853679/helm-templating-doesnt-let-me-use-dash-in-names